src/smefit/loader.py

```diff
@@ -545,11 +545,11 @@ def load_datasets(
     commondata_path,
     datasets,
     operators_to_keep,
-    order,
     use_quad,
     use_theory_covmat,
     use_t0,
     use_multiplicative_prescription,
+    default_order="LO",
```

Comment:
why do you need to have a default "LO" ?  Do you need to load a dataset without any theory being loaded?

Reply:
This is just the default theory order that gets loaded if one doesn't specify it for a particular dataset. Of course, the default can be set to NLO as well, but not all predictions are available at NLO.

Reply:
sure, but sorry to be picky why do you want to have a default ? 
I mean if you load, you need to specify an order